bentoml/_internal/bento/bento.py

```diff
@@ -32,10 +32,14 @@
 BENTO_YAML_FILENAME = "bento.yaml"
 
 
-@attr.define(repr=False)
+@dataclass
```

Comment:
@sauyon hi sauyon. just curious why we changed to dataclass here?

Reply:
Sorry, I missed this---I'll change it back, it's because version 20 of attrs doesn't support type checking; I'm going to just force attrs 21+ for dev-dependencies so that everything type-checks nicely.